test-requirements.txt

```diff
@@ -1,11 +1,7 @@
-six
-pyOpenSSL
-flake8
-flake8-mutable
-flake8-print
-pylint
-setuptools-lint
+# General Deps
```

Comment:
Agreed, I just don't know of a way to reconcile having ansible in requirements.txt, but also being able to template it in tox.